cli/pcluster/configure/easyconfig.py

```diff
@@ -14,6 +14,7 @@
 
 from pcluster.cluster_model import ClusterModel
 from pcluster.config.hit_converter import HitConverter
+from pcluster.config.validators import HEADNODE_UNSUPPORTED_INSTANCE_TYPES
```

Comment:
Not related to your patch but could you please rename HEADNODE_UNSUPPORTED... to HEAD_NODE_UNSUPPORTED  to be aligned with the naming convention we're using elsewhere.